mint.json

```diff
@@ -422,6 +423,7 @@
           "group": "Process data",
           "pages": [
             "processing/overview",
+            "processing/sql-references",
```

Comment:
put it under "sql"?

Reply:
> put it under "sql"?

Updated, but putting the auto-direct page like this as the first item under a section doesn't work well, so I moved it to the last one under SQL section
<img width="406" alt="image" src="https://github.com/user-attachments/assets/d44f25ae-4d5f-456f-9c4c-4a3bdcb6aad1">